code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -947,7 +951,11 @@
 
 /mob/living/silicon/robot/updatehealth()
 	..()
-	if(health < maxHealth*0.5) //Gradual break down of modules as more damage is sustained
+	var/missingHealth = 200 - (health + 100)
```

Comment:
Should be calculated based on maxhealth

Reply:
I'm not sure how to go about this because maxhealth is not actually how much health a cyborg has. Cyborgs have 200 HP before breaking, but when fully repaired maxhealth is only 100, which horribly throws off calculations. 

I risk a divide by zero if I just use maxhealth instead of creating my own "missing health" variable.

Reply:
This is:
200 - (health + 100)
Which is the same as
200 - health - 100
Which is the same as
100 - health
Which is the same as
`maxhealth - health` anyway
This kind of sounds weird (I had to think about it for a bit), but if health goes negative, then the subtraction will add, increasing the value of missingHealth.